sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,369 @@
+# December'22 release notes
+
+Release notes for commit range [`0f579bae..6977f1ac`](https://github.com/intel/llvm/compare/0f579bae...6977f1ac)
+
+## New features
+
+### SYCL Compiler
+
+- Added support for per-object device code compilation under the option
+`-fno-sycl-rdc`. This improves compiler performance and reduces memory usage,
+but can only be used if there are no cross-object dependencies. [f884993d]
+- Added support for per-aspect device code split mode. [9a2c4fe5]
+- Added GPU specific device targets for ahead-of-time compilation. [5bd5c871]
+- Extended support for the large GRF mode to non-ESIMD kernels. [9994934b]
+[ab2a42c6]
+- Added support for the `sycl_ext_oneapi_kernel_properties` experimental
+extension. [332e4ee0] [27454de6] [70ee3d56] [430c7221]
+- Added support for AMD and NVIDIA architectures as arguments for
+`-fsycl-targets`. [e5de913f]
```

Comment:
Not needed anymore, part of sycl_ext_intel_device_architecture
```suggestion
```

Reply:
Or, to be more clear:
```
- Extended `sycl_ext_intel_device_architecture` with support for AMD and NVIDIA architectures. [e5de913f]
```
In this case, remove [e5de913f] from my suggestion below (https://github.com/intel/llvm/pull/8154#discussion_r1091993815)